user/deployment/npm.md

```diff
@@ -132,3 +132,36 @@ after_deploy:
   - ./after_deploy_1.sh
   - ./after_deploy_2.sh
 ```
+
+### Troubleshooting common problems
+
+#### Skipping a deployment with the npm provider because this is not a tagged commit
```

Comment:
I am not sure if you want to characterize these things as "problems". There are very good reasons to specifically limit deployment to a single job, and calling these are "problems" is misleading.

Conditional deployment is already covered in https://docs.travis-ci.com/user/deployment/#Conditional-Releases-with-on%3A.